linebot/find_dollar_sign.go

```diff
@@ -0,0 +1,20 @@
+package linebot
+
+import "unicode/utf16"
+
+/*
+When you want to send a text message with emoji, you need to add
+$ in the text, and identify the index of the $ in the text.
+
+FindDollarSignIndexInUni16Text helps you to find the index of the $ in the text.
+*/
+func FindDollarSignIndexInUni16Text(text string) (indexes []int32) {
+	bytes := utf16.Encode([]rune(text))
+	for i := range bytes {
+
+		if bytes[i] == 36 {
```

Comment:
To clarify 36 is `$`, how about this? 
```suggestion
    encoded := utf16.Encode([]rune(text))
    for i, unit := range encoded {
        if unit == '$' {
```

Reply:
The value unit itself is actually value 36. Because the value has been encoded.

But I see your point. So I will do this instead:

```go

type CharInUTF16 uint16

const dollarSign CharInUTF16 = 36

...

encoded := utf16.Encode([]rune(text))
	for i, unit := range encoded {

		if unit == uint16(dollarSign) {

```

Reply:
thanks!